modules/swagger-codegen/src/main/resources/r/model.mustache

```diff
@@ -76,6 +76,22 @@
         {{/isListContainer}}
         {{/vars}}
       )
+    },
+    fromJSON = function({{classname}}Json) {
+      {{classname}}Object <- jsonlite::fromJSON({{classname}}Json)
+      {{#vars}}
+      {{#isPrimitiveType}}
+      self${{name}} <- {{classname}}Object${{name}}
+      {{/isPrimitiveType}}
+      {{^isPrimitiveType}}
+      {{#isListContainer}}
+      self${{name}} <- lapply({{classname}}Object${{name}}, function(x) {{datatype}}$new()$fromJSON(x)
```

Comment:
Here, you are passing a data frame to fromJSON method. You should pass a JSON string like this : 

 self${{name}} <- lapply({{classname}}Object${{name}}, function(x) {{datatype}}$new()$fromJSON(jsonlite::toJSON(x))